app/controllers/movie_controller.rb

```diff
@@ -1,40 +1,17 @@
 class MovieController < ApplicationController
   def top_forty
-    top_40 = []
-    page = 1
-    2.times do
-      response = Faraday.get("https://api.themoviedb.org/3/discover/movie?api_key=#{ENV["MOVIE_API_KEY"]}&language=en-US&sort_by=popularity.desc&include_adult=false&include_video=false&page=#{page}")
-      json = JSON.parse(response.body, symbolize_names: true)
-      top_40 << json[:results]
-      page += 1
-    end
-    @results = top_40.flatten.sort_by do |result|
-      result[:vote_average]
-    end.reverse
+    @movies = MovieFacade.top_forty
```

Comment:
Just getting to see this "large" method turned into a single line... wow. Super impressive! Excited to see the rest of this refactor!